### PR TITLE
non-cdo classifications set up

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1665)

### CODE REVIEW

File: talentmap_api/fsbid/services/classifications.py [hunk 1]
```diff
@@ -57,3 +57,17 @@ def delete_client_classification(jwt_token=None, perdet_seq_num=None, data=None)
...
 
     return fsbid_classifications_to_tmap(response.get("Data", {}))
+
+def get_user_classifications(jwt_token=None, perdet_seq_num=None):
+    '''
+    Get the user's classification(s)
+    '''
+    from talentmap_api.fsbid.services.client import fsbid_classifications_to_tmap
+    url = f"{BTP_ROOT}?request_params.perdet_seq_num={perdet_seq_num}"
+    print(url)
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/classifications.py:67]: needed?

File: talentmap_api/fsbid/services/classifications.py [hunk 2]
```diff
@@ -57,3 +57,17 @@ def delete_client_classification(jwt_token=None, perdet_seq_num=None, data=None)
         return None
 
     return fsbid_classifications_to_tmap(response.get("Data", {}))
+
+def get_user_classifications(jwt_token=None, perdet_seq_num=None):
+    '''
+    Get the user's classification(s)
+    '''
+    from talentmap_api.fsbid.services.client import fsbid_classifications_to_tmap
+    url = f"{BTP_ROOT}?request_params.perdet_seq_num={perdet_seq_num}"
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/classifications.py:66]: accessible by users with only `fsofficer`?

**Comment by mjoyce91** [comment]
Wendy doesn't have the bidder role, so should we even show classifications? Would non-bidders ever have any?

Screen Shot 2021-08-09 at 3 26 08 PM

File: talentmap_api/fsbid/services/classifications.py (hunk 2 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/classifications.py:66]: Nah

File: talentmap_api/fsbid/services/classifications.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/classifications.py:67]: 🤷‍♀️ never know

File: talentmap_api/fsbid/views/classifications.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import coreapi
+
+from rest_framework.response import Response
+from rest_framework.schemas import AutoSchema
+
+from talentmap_api.fsbid.views.base import BaseView
+import talentmap_api.fsbid.services.classifications as classifications_services
+
+
+class FSBidClassificationsView(BaseView):
```
Comment (mjoyce91) [inline talentmap_api/fsbid/views/classifications.py:10]: Can we add

```py
permission_classes = (IsAuthenticated,)
```

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/views/classifications.py:10]: Nah. Jk done!

File: talentmap_api/fsbid/services/reference.py [hunk 4]
```diff
@@ -141,12 +142,31 @@ def fsbid_locations_to_talentmap_locations(data):
...
+        '6': '6/8 Rule',
+        'F': 'Fair Share',
+        'C': 'Critical Need Language',
+        'C1': 'Critical Need Language 1st Tour Complete',
+        'CC': 'Critical Need Language Final Tour Complete',
+        'R': 'Recommended for Tenure',
+        'A': 'AMB Ambassador',
+        'F1': 'Pickering Fellows',
+        'F2': 'Rangel Fellows',
+        'P': 'Pickering Fellow',
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/reference.py:160]: Why is F1 plural but P not?

**Pull request merged**